### ISSUE TITLE
CVE-2023-6246 vulnerability with latest docker image.

### ISSUE THREAD

**Issue by xllalitha** [locustio/locust#2581]
### Prerequisites

- [X] I am using [the latest version of Locust](https://github.com/locustio/locust/releases/)
- [X] I am reporting a bug, not asking a question

### Description

CVE-2023-6246 vulnerability (High Severity) exists with latest docker image. Vulnerability on glibc package. 

Description
A heap-based buffer overflow was found in the __vsyslog_internal function of the glibc library. This function is called by the syslog and vsyslog functions. This issue occurs when the openlog function was not called, or called with the ident argument set to NULL, and the program name (the basename of argv[0]) is bigger than 1024 bytes, resulting in an application crash or local privilege escalation. This issue affects glibc 2.36 and newer.

### Command line

 locust -f mylocustfile.py -t 10s --headless

### Locustfile contents

```python3
import time
from locust import HttpUser, task, between

class QuickstartUser(HttpUser):
    wait_time = between(1, 5)

    @task
    def hello_world(self):
        self.client.get("/hello")
        self.client.get("/world")

    @task(3)
    def view_items(self):
        for item_id in range(10):
            self.client.get(f"/item?id={item_id}", name="/item")
            time.sleep(1)

    def on_start(self):
        self.client.post("/login", json={"username":"foo", "password":"bar"})
```

### Python version

python3

### Locust version

latest

### Operating system

locust docker image.

**Comment by cyberw** [comment]
Please provide some details on how this could impact locust? If it requires shell access it probably isnt very relevant.

**Comment by cyberw** [comment]
`A heap-based buffer overflow was found in the __vsyslog_internal function of the glibc library. This function is called by the syslog and vsyslog functions. This issue occurs when the openlog function was not called, or called with the ident argument set to NULL, and the program name (the basename of argv[0]) is bigger than 1024 bytes, resulting in an application crash or local privilege escalation. This issue affects glibc 2.36 and newer.`

The locust image is not for running general purpose code, so being able to achieve local privilege escalation by running specific code is not really relevant. Once an updated glibc is available in the parent image (python:3.11-slim), we'll get it automatically.
